packages/ra-ui-materialui/src/detail/EditView.tsx

```diff
@@ -89,7 +90,7 @@ export const EditView = (props: EditViewProps) => {
                             resource,
                             save,
                             saving,
-                            undoable,
+                            undoable: undoable || mutationMode === 'undoable',
```

Comment:
Shouldn't we pass `mutationMode` as well ?

Reply:
yep, done